addons/website_event_exhibitor/static/src/js/event_exhibitor_connect.js

```diff
@@ -31,6 +31,11 @@ publicWidget.registry.eventExhibitorConnect = publicWidget.Widget.extend({
         });
     },
 
+    destroy () {
+        this._super(...arguments);
+        this.el.removeEventListener("click", this._onConnectClick.bind(this));
```

Comment:
This would remove a new bound version of `this._onConnectClick`, not th added one. You must keep track of the results of `.bind()` to be able to remove it.

Reply:
@bso-odoo  I see i must first store the reference, and then remove the event listener using the same reference. 

something like this right ?
```js
   // while binding eventlistner:
           this._boundOnConnectClick = this._onConnectClick.bind(this);
        // Add the event listener using the stored reference
        this.el.addEventListener("click", this._boundOnConnectClick);

// then in destroy i should remove the refrenced one 
 this.el.removeEventListener("click", this._boundOnConnectClick);
```

Reply:
Why not binding it right away upon declaration ?
```diff
-        this._onConnectClick = debounce(this._onConnectClick, 500, true);
+        this._onConnectClick = debounce(this._onConnectClick, 500, true).bind(this);
```